### DISCUSSION TITLE
Support for reMarkable Paper Pro?

### DISCUSSION THREAD

**Discussion by githubtefo** [toltec-dev/toltec#910] [Ideas]
Any plans for future support on this model?
I just bought it, so happy to contribute as a user on those efforts.
Thanks

**Comment by Eeems** [comment]
More information is needed about the tech stack on the rM  pro before we can work on supporting it. None of the maintainers have one currently, and it will be a little bit before any does. When a maintainer does get the device, and we can start looking further into what needs to be done to support it, we will work on adding support to toltec.

Current things I believe will be of concern:

- New display technology will require a new display server for backwards compatibility with existing apps
- Toltec will need to support OS 3.14+ as that's what the rM pro ships with, so we'll need to work on moving that direction
- New system layout will require work to get current toltec installation process to support the new device. There are now overlay filesystems etc that may impact things.

**Reply by davux** [reply]
Also I think the architecture is different. See #929.

**Comment by githubtefo** [comment]
I see - I guess I would need to pay for Connect for a while before being able to mount webdav files from my server then.
Luckily, this version has encryption by default.
Thanks and looking forward to having toltec on my new reMarkable soon!

**Reply by Eeems** [reply]
You could look into getting 3.14 support into rmfakecloud, and setting that up, that way you would be able to use it's webdav integration.

**Reply by githubtefo** [reply]
Thanks for the input!
So far it is on v3.13 so hopefully soon will be available for 3.14!

**Comment by davux** [comment]
Happy to help testing as needed too!

**Comment by rootplease** [comment]
Is there any recommendations on where someone with software engineering experience can get involved / acquire the needed background knowledge or general pointers for trying to get toltec to work on my rmp?

**Comment by aneth-dev** [comment]
Is there any update?

**Reply by Eeems** [reply]
All activity on the repository is public. You can see if there have been commits on other branches or new PRs.
